### PR TITLE
hamsterdb 2.1.10

### PR DESCRIPTION
**Pull request by dunn**
Is this something to worry about?
```
Warning: Could not fix libuv.dylib in /usr/local/Cellar/hamsterdb/2.1.10/bin/ham_bench
Warning: Could not fix libuv.dylib in /usr/local/Cellar/hamsterdb/2.1.10/bin/hamzilla
Warning: Could not fix libuv.dylib in /usr/local/Cellar/hamsterdb/2.1.10/lib/libhamserver.0.dylib
Warning: Could not fix libuv.dylib in /usr/local/Cellar/hamsterdb/2.1.10/share/samples/.libs/client1
Warning: Could not fix libuv.dylib in /usr/local/Cellar/hamsterdb/2.1.10/share/samples/.libs/server1
```

### CODE REVIEW

**Comment by mikemcquaid** [comment]
Failing 10.8 with Mountain Lion compiler issues I suspect. Roll on 10.11 :wink: :http://bot.brew.sh/job/Homebrew%20Pull%20Requests/24913/version=mountain_lion/testReport/junit/brew-test-bot/mountain_lion/install_hamsterdb/

**Comment by dunn** [comment]
`fails_with` clang there or maybe just a minimum macos requirement?

**Comment by mikemcquaid** [comment]
Worth trying the `fails_with` approach first, I think.

**Comment by dunn** [comment]
Should I bump the vendored libuv while I'm at it?  They say [up to 0.11.22 will work](https://github.com/cruppstahl/hamsterdb/commit/79bc5e96db10a6633ab50bb4e30146bb6580d72f), but the pre-1.0.0 versioning scheme is confusing so I'm not sure if that's a development release or not.

**Comment by mikemcquaid** [comment]
Yeh, maybe worth a try.

**Comment by dunn** [comment]
Ha, `fails_with :clang` is already in there, ML was building with llvm.  I'll try:
```
  if MacOS.version <= :mountain_lion
    # "error: member access into incomplete type 'const std::type_info'"
    fails_with :clang
    fails_with :llvm    
  end
```

**Comment by mikemcquaid** [comment]
Would be good to still use a `build` and add separate `fails_with` there, just for niceness. If this is a CXX11 thing we can use `needs :cxx11`.

**Comment by dunn** [comment]
OK I'll try cxx11 first.

**Comment by dunn** [comment]
Ack. Finally got it passing then remembered to bump libuv and the bots choked I think.

**Comment by mikemcquaid** [comment]
Rebuilding.

File: Library/Formula/hamsterdb.rb [hunk 1]
```diff
@@ -68,16 +65,19 @@ def install
...
-    system "./configure", "--disable-debug", "--disable-dependency-tracking",
-                          "--prefix=#{prefix}",
-                          *features
+    system "./configure", *args
     system "make", "install"
+
+    share.install "samples"
   end
 
   test do
```
Comment (asparagui) [inline Library/Formula/hamsterdb.rb]: :ocean:

**Comment by asparagui** [comment]
Getting the same warnings per above, will defer this one.

**Comment by mikemcquaid** [comment]
Thanks @dunn!

**Pull request closed without merging**